docs/src/iterators.md

```diff
@@ -28,28 +28,42 @@ jacobi_iterable(x, A::SparseMatrixCSC, b; maxiter::Int = 10) =
 
 Now if you apply Jacobi iteration multiple times with the same matrix for just a few iterations, it makes sense to initialize the iterable only once and reuse it afterwards:
 
-```julia
-A = sprand(10_000, 10_000, 10 / 10_000) + 20I
-b1 = rand(10_000)
-b2 = rand(10_000)
-x = rand(10_000)
+```jldoctest
```

Comment:
I'm not 100% sure whether it's a great idea to run a doctest with a problem size of 10_000. Also this is not really meant as a test, but rather as an example.

Reply:
I modified the problem to be much smaller and suitable for CI doctests.

> I'm fine running it to ensure it doesn't break, but I don't think it should be treated as a test.

How would you do that?

Reply:
Perfect! As is is good, it's basically just a smoke test (if it doesn't throw an error, it's fine).